### PR TITLE
Add repr for Llamaindex UnitycatalogTool

### PR DESCRIPTION
**Pull request by serena-ruan**

### CODE REVIEW

File: integrations/llama_index/src/ucai_llamaindex/toolkit.py [hunk 1]
```diff
@@ -41,6 +42,13 @@ def __init__(
         super().__init__(*args, fn=fn, metadata=metadata, **kwargs)
         self.client_config = client_config
 
+    def __repr__(self) -> str:
+        return (
+            "UnityCatalogTool("
+            + ", ".join([f"{k}={v}" for k, v in asdict(self.metadata).items() if v])
```
Comment (harupy) [inline integrations/llama_index/src/ucai_llamaindex/toolkit.py]: why `if v`? Should we ignore an empty string

Comment (harupy) [inline integrations/llama_index/src/ucai_llamaindex/toolkit.py]:
```suggestion
            + ", ".join([f"{k}={v!r}" for k, v in asdict(self.metadata).items() if v])
```

to quote string values.

Comment (harupy) [inline integrations/llama_index/src/ucai_llamaindex/toolkit.py]:
```suggestion
            + ", ".join(f"{k}={v}" for k, v in asdict(self.metadata).items() if v)
```

to skip temporary list creation.

Comment (serena-ruan) [reply integrations/llama_index/src/ucai_llamaindex/toolkit.py]: We can include empty string :) Let me update

**Review by harupy — APPROVED** [review]
LGTM

**Pull request merged**